Report.md

```diff
@@ -73,6 +73,28 @@ Design Thinking Approach
+
+### Empathise
+Phrase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives the tool of an empathy map was utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 
+
+### Define
+Phrase two of defining allowed us to synthesis the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This staged allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given. 
+
+### Ideate
+Phrase three consisted of utilising the user research conducted to ideate solutions to the problem statements we created in the prior stage. The brainstorming and worst possible idea techniques were utilised to assist within our ideation stage. These techniques were chosen to initiate freeform group discussions and simulated creative solutions in light of the problems identified. These sessions consisted of developing “how might we? (HMW)” statements and deriving features relative to the point of view statements developed in the define stage. The conclusions of the brainstormed were documented in (need to do this???). At the conclusion of this stage, the agreed features were converted to issues to be developed for the prototype and testing stage. 
+
+### Prototype/Test
+The prototype and subsequent testing phrase is representative of the final stages within the design thinking approach. This consists of producing a small-scaled, mock-up of the product or specific features found within the product so that tangible investigations can occur on the problem solutions generated in the previous stage. To assist in the prototype stage the team designed artefacts representative of the design of the system and the user flow these are demonstrated within the use case diagram, activity diagrams, class diagram and collaborative diagrams provided. Due to the nature of this project, the prototype and respective testing are not within scope. 
```

Comment:
"testing phase" ahahaha.
Hm also personally think just "investigations can occur on the solutions generated" rather than 'problem solutions' so it's just a little slimmer.
"design of the system and the user flow. These are demonstrated" kind of a run-on sentence, maybe split it into two?